packages/thirdweb/src/utils/domains.ts

```diff
@@ -24,20 +24,27 @@
    * @default "bundler.thirdweb.com"
    */
   bundler?: string;
+  /**
+   * The base URL for the chainsaw server.
+   * @default "chainsaw.thirdweb.com"
+   */
+  chainsaw?: string;
 };
 
 export const DEFAULT_RPC_URL = "rpc.thirdweb.com";
 const DEFAULT_IN_APP_WALLET_URL = "embedded-wallet.thirdweb.com";
 const DEFAULT_PAY_URL = "pay.thirdweb.com";
 const DEFAULT_STORAGE_URL = "storage.thirdweb.com";
 const DEFAULT_BUNDLER_URL = "bundler.thirdweb.com";
+const DEFAULT_CHAINSAW_URL = "https://chainsaw.thirdweb.com";
```

Comment:
Remove scheme

Reply:
Done in [2f7bc1b](https://github.com/thirdweb-dev/js/pull/3741/commits/2f7bc1bfd525b8c6a4cbcb50ae91fb8fce66a338)